analyzers/tests/SonarAnalyzer.UnitTest/SonarAnalyzer.UnitTest.csproj

```diff
@@ -14,6 +14,7 @@
       -->
     <AssetTargetFallback>$(AssetTargetFallback);net40-client;portable-net45+win8+wp8+wpa81;</AssetTargetFallback>
     <ProjectGuid>{e11606ca-a186-4fee-ba30-b1688747cd1a}</ProjectGuid>
+    <LangVersion>11</LangVersion>
```

Comment:
This isn't mandatory yet, right? The whole solution should have an unified version (defined [here](https://github.com/SonarSource/sonar-dotnet/blob/94efcb5db4f9312679b4351e7ddb4e05aa6d5387/analyzers/Analyzers.targets#L8)) and we should bump it soon after the public release.

Reply:
This is not mandatory but using these features in our code can help us better understand them and their impact. Since we will use them only on test code it will avoid any risk for the released analyzer.

Reply:
We should not do that. Having different versions in UTs and production is causing pain. And leads to inconsistencies that took some time to clean up.

We should update as soon as the public release is out, for the whole codebase (while making sure it works first, of course)

Reply:
> is causing pain. And leads to inconsistencies that took some time to clean up.

can you give an example?

Reply:
Getting used to a feature in UTs (like `??=`) and being unable to use it in other code is painful.